### ISSUE TITLE
Adding lambda trigger function says there are no DynamoDB resources configured?

### ISSUE THREAD

**Issue by helloniklas** [aws-amplify/amplify-cli#3914]
**Which Category is your question related to?**

amplify add function

**Amplify CLI Version** 
4.18.0

**Provide additional details e.g. code snippets**
I've added auth and api. It created the dynamo db table etc in the cloud. The AWS App sync queries are running fine. When I then try and do amplify add function for a lambda trigger. And select 

`Choose a DynamoDB event source option Use storage category DynamoDB table configured in the current Amplify project
`
It just says: There are no DynamoDB resources configured in your project currently

But the tables are there in the cloud. When I done this previously it worked fine and I got present a list of the tables to choose from? 

Not sure why it thinks there are no resources there?

**Comment by CharanRoot** [comment]
there is already opened ticket for similar feature 
https://github.com/aws-amplify/amplify-cli/issues/2203

**Comment by edwardfoyle** [comment]
Hi @helloniklas are the dynamodb tables you are trying to access the ones generated by the @model annotations in your API schema? If so, you need to select the following options in the `amplify add function` flow:
```
Choose the function runtime that you want to use: NodeJS
? Choose the function template that you want to use: Lambda trigger
? What event source do you want to associate with Lambda trigger? Amazon Dyn
amoDB Stream
? Choose a DynamoDB event source option Use API category graphql @model back
ed DynamoDB table(s) in the current Amplify project
```
Then you should be prompted to chose your API and then chose the specific model table you want your lambda to be triggered off of. Let me know if that's what you're looking for!

**Comment by helloniklas** [comment]
@edwardfoyle hm that works nicely now... not sure why this didn't work before... thanks.

One other thing though. It asks if I want to edit the local lambda function now. If I say yes and edit the file trigger-dynamodb.js. This gets uploaded when I push. But, the lambda is not set to use this function by default but rather then index.js file... So I still manually have to edit this in the lambda console? Shoulnd't it just point to the trigger-dynamodb.js by default?

**Comment by edwardfoyle** [comment]
@helloniklas Just tried it out and looks like that behavior is a bug. The template creates `src/index.js` which is correct but the editor dialog opens `src/trigger-dynamodb.js` which is incorrect. While we're working on a fix, you can get around the issue by opening `src/index.js` manually and editing your trigger from there.

**Comment by edwardfoyle** [comment]
@helloniklas This should be fixed in the latest release of the CLI (4.19.0)
